### PR TITLE
Add new command `morph list-containers` in `neofs-adm`

### LINKED ISSUE #1689: Add ability to list all containers in the system
Now administrators of neofs installation don't have ability to show full list of containers in the system for audit of another aim. We can use --owner key in neofs-cli in case then we know all containers owner but it doesn't work in case when we create containers with -g key creating key on the fly.

### PR DESCRIPTION
**Pull request by acid-ant**
Close #1689

Signed-off-by: Anton Nikiforov <an.nikiforov@yadro.com>

Here is an example output:
```
@:~/workspace/neofs-node$ neofs-adm morph list-containers --rpc-endpoint http://morph-chain.neofs.devenv:30333
CKWBSDN6BBQUWqe9SJgNQ13YJegMCUMEjfWXrM6kD8Mi
D2CpK1zvwgdF9EQGU7G9hVQchWJZm6EhsKDgWJYcr6xV
GF1pXsLkBkj4Bp3KPay6wgZLssSa5RF3b9XZRK5kCtRp
JDuhym3ZdL8RH4WwM3LeYr9Ung37sKKtJcYnrupC8p44
@:~/workspace/neofs-node$ 
@:~/workspace/neofs-node$ neofs-cli container list --rpc-endpoint 192.168.130.71:8080 --wallet workspace/neofs-dev-env/wallets/wallet.json
Enter password > 
GF1pXsLkBkj4Bp3KPay6wgZLssSa5RF3b9XZRK5kCtRp
JDuhym3ZdL8RH4WwM3LeYr9Ung37sKKtJcYnrupC8p44
@:~/workspace/neofs-node$ 
```

### CODE REVIEW

File: cmd/neofs-adm/internal/modules/morph/container.go [hunk 1]
```diff
@@ -104,6 +104,72 @@ func dumpContainers(cmd *cobra.Command, _ []string) error {
 	return os.WriteFile(filename, out, 0o660)
 }
 
+func listContainers(cmd *cobra.Command, _ []string) error {
+	filename, err := cmd.Flags().GetString(containerDumpFlag)
```
Comment (fyrchik) [inline cmd/neofs-adm/internal/modules/morph/container.go:108]: What do we need this for?

File: cmd/neofs-adm/internal/modules/morph/container.go [hunk 2]
```diff
@@ -104,6 +104,72 @@ func dumpContainers(cmd *cobra.Command, _ []string) error {
...
+
+	inv := invoker.New(c, nil)
+
+	nnsCs, err := c.GetContractStateByID(1)
+	if err != nil {
+		return fmt.Errorf("can't get NNS contract state: %w", err)
+	}
+
+	var ch util.Uint160
+	s, err := cmd.Flags().GetString(containerContractFlag)
```
Comment (fyrchik) [inline cmd/neofs-adm/internal/modules/morph/container.go:126]: Missing flag returns default value, `err` here is for developer errors, or am I mistaken?

File: cmd/neofs-adm/internal/modules/morph/container.go (hunk 1 above)
Comment (acid-ant) [reply cmd/neofs-adm/internal/modules/morph/container.go]: Removed as discussed.

File: cmd/neofs-adm/internal/modules/morph/container.go [hunk 3]
```diff
@@ -104,6 +104,52 @@ func dumpContainers(cmd *cobra.Command, _ []string) error {
...
+			return fmt.Errorf("unable to decode container contract hash from arg %s: %w", containerContractFlag, err)
+		}
+	}
+
+	cids, err := unwrap.ArrayOfBytes(inv.Call(ch, "list", ""))
+	if err != nil {
+		return fmt.Errorf("%w: %v", errInvalidContainerResponse, err)
+	}
+
+	if len(cids) > 0 {
```
Comment (fyrchik) [inline cmd/neofs-adm/internal/modules/morph/container.go:140]: We have only a loop inside which works fine. Do we need this condition?

**Review by fyrchik — APPROVED** [review]

File: cmd/neofs-adm/internal/modules/morph/container.go (hunk 3 above)
Comment (acid-ant) [reply cmd/neofs-adm/internal/modules/morph/container.go]: You right, redundant, forget to remove. Updated.

File: cmd/neofs-adm/internal/modules/morph/container.go [hunk 4]
```diff
@@ -104,6 +104,50 @@ func dumpContainers(cmd *cobra.Command, _ []string) error {
...
+		return fmt.Errorf("%w: %v", errInvalidContainerResponse, err)
+	}
+
+	for _, id := range cids {
+		var idCnr cid.ID
+		err = idCnr.Decode(id)
+		if err != nil {
+			return fmt.Errorf("unable to decode container id: %w", err)
+		}
+		cmd.Printf("%s\n", idCnr)
```
Comment (cthulhu-rider) [inline cmd/neofs-adm/internal/modules/morph/container.go:146]: (minor)
```suggestion
		cmd.Println(idCnr)
```

**Review by cthulhu-rider — APPROVED** [review]

File: cmd/neofs-adm/internal/modules/morph/container.go (hunk 4 above)
Comment (acid-ant) [reply cmd/neofs-adm/internal/modules/morph/container.go]: Thanks, done

File: cmd/neofs-adm/internal/modules/morph/container.go [hunk 5]
```diff
@@ -22,6 +22,37 @@ import (
 
 var errInvalidContainerResponse = errors.New("invalid response from container contract")
 
+func getContractHash(cmd *cobra.Command, inv *invoker.Invoker, c Client) (util.Uint160, error) {
```
Comment (fyrchik) [inline cmd/neofs-adm/internal/modules/morph/container.go:25]: Can we name this `getContainerContractHash` at least?

Comment (acid-ant) [reply cmd/neofs-adm/internal/modules/morph/container.go]: Renamed.

**Review by fyrchik — APPROVED** [review]

**Review by fyrchik — APPROVED** [review]

**Pull request merged**